variants/aws-ecs-2/Cargo.toml

```diff
@@ -0,0 +1,50 @@
+[package]
+name = "aws-ecs-2"
+version = "0.1.0"
+edition = "2021"
+publish = false
+build = "../build.rs"
+# Don't rebuild crate just because of changes to README.
+exclude = ["README.md"]
+
+[package.metadata.build-variant.image-features]
+grub-set-private-var = true
```

Comment:
This means we can safely add `BootSettings` / `settings.boot` support to these  new variants.